### PR TITLE
API-5473 - Updates to nod pdf

### PR DESCRIPTION
**Pull request by nathanbwright**
## Description of change
Updates to the NOD pdf (based on feedback from the board).

## Original issue(s)
https://vajira.max.gov/browse/API-5473

### CODE REVIEW

File: modules/appeals_api/app/services/appeals_api/pdf_construction/generator.rb [hunk 1]
```diff
@@ -63,16 +63,7 @@ def finalize_pages
       end
 
       def stamp
-        stamper = CentralMail::DatestampPdf.new(@unstamped_path)
```
Comment (yakschuss) [inline modules/appeals_api/app/services/appeals_api/pdf_construction/generator.rb:66]: I'm confused by this change. Is this stamp no longer needed in the NOD PDF?

**Review by yakschuss — APPROVED** [review]
Looks good to me! I might have missed it, but where do we handle the 'Move the SSN stamp to the middle' AC? I see that we've moved the VA-GOV timestamp location.

**Comment by nathanbwright** [comment]
> Looks good to me! I might have missed it, but where do we handle the 'Move the SSN stamp to the middle' AC? I see that we've moved the VA-GOV timestamp location.

It's AC # 2 https://vajira.max.gov/browse/API-5473

**Comment by yakschuss** [comment]
> > Looks good to me! I might have missed it, but where do we handle the 'Move the SSN stamp to the middle' AC? I see that we've moved the VA-GOV timestamp location.
> 
> It's AC # 2 https://vajira.max.gov/browse/API-5473

Right, I must have missed it in the code.

**Review by cilestin — APPROVED** [review]
LGTM!

**Pull request merged**